docs/content/users/extend/customizing-images.md

```diff
@@ -88,14 +88,15 @@ The following environment variables are available for the web Dockerfile to use
 * `$username`: the username inferred from your host-side username
 * `$uid`: the user ID inferred from your host-side user ID
 * `$gid`: the group ID inferred from your host-side group ID
-* `$DDEV_PHP_VERSION`: the PHP version declared in your project configuration
+* `$DDEV_PHP_VERSION`: the PHP version declared in your project configuration (provided in versions after v1.21.4)
```

Comment:
Does this say the same thing in fewer words?

```suggestion
* `$DDEV_PHP_VERSION`: the PHP version declared in your project configuration (v1.21.4 or higher)
```

Reply:
The problem is that it's *NOT* in v1.21.4, will be in next release. It's the "latest" docs problem that let someone astray in #4497

Reply:
Sorry, bad reading on my part! Could that be `v1.24.5 or higher` then? (Or whatever the next release is.)

Reply:
I usually am unwilling to commit to release type (minor vs patch) this far in advance :) That's the problem.

Reply:
Makes sense! I guess `(after v1.21.4)` is best then for brevity, though the reader will still have to figure out what the actual minimum version is. Sounds like there’s no way to avoid that.

Reply:
[mkdocs material](https://squidfunk.github.io/mkdocs-material/) (which we use) has a capability of marking releases when showing features. It's nice. Might be overkill, but sure would help here.

Reply:
I’ve seen these little markers and I like them:
![Screen Shot 2022-12-27 at 03 52 38 PM@2x](https://user-images.githubusercontent.com/2488775/209737439-d67808db-e154-41ba-b63d-5ef7e4516dea.png)

But.

1. It depends on a layout more like our config and command reference where each thing (property, command, environment variable) has its own heading.
2. I think it’d get a bit _more_ confusing to use these in conjunction with the version selector that’s already down there in the bottom right corner.

Reply:
Agreed on both points. Just chatting away about this because we've talked about it before and about this exact confusion point before.

Reply:
🤦‍♂️